Documentation/Modules.md

```diff
@@ -0,0 +1,121 @@
+# Modules
+
+The SourceKit-LSP package contains the following non-testing modules.
```

Comment:
What's the "non-testing" mean here?

Reply:
Technically the `testTargets` are also modules. But I didn’t include them here because they aren’t really relevant for the modules structure.

Reply:
Ah I see 👍 Not sure why this didn't make me think of `testTarget`.